postoffice_django/config.py

```diff
@@ -23,7 +23,8 @@ def configure_publishers() -> None:
             uncreated_publishers.append(consumer)
 
     if uncreated_publishers:
-        raise BadPublisherCreation(uncreated_publishers)
```

Comment:
If we aren't using this exception anymore, can you remove this please?

Reply:
Sure!

Reply:
I'm gonna remove the BadTopicCreation as well

Reply:
Done!